### DISCUSSION TITLE
iOS: Add favorites via URL

### DISCUSSION THREAD

**Discussion by mcritz** [community/community#3610] [Mobile]
I'd like to add favorites by URL. 

Ex: 

A search for "https://github.com/hummingbird-project/hummingbird" should show the matching repo in the search results.

**Comment by PranamBhat** [comment]
Hi @mcritz 

You should search like this "[github.com/hummingbird-project/hummingbird](github.com/hummingbird-project/hummingbird)"

**Reply by mcritz** [reply]
That doesn’t work as expected. Searching for a favorite using that pattern does not find the canonical source. Also, even if it actually worked, that’s not the feature I’m asking for. I want to paste the entire URL as I might copy from an email or chat message.

**Comment by PranamBhat** [comment]
@mcritz 

GitHub already mentioned "/" in the "Search Box". Which means, search the keyword after "https://".

![image](https://user-images.githubusercontent.com/32120033/113571535-2e0cf900-9634-11eb-8424-cae674dc7a7a.png)

**Reply by mcritz** [reply]
Perhaps I’m missing where this UI appears in the native iOS client, but it looks like the responsive web version of GitHub.com and not the GitHub app you download from the App Store. Please correct me if I’m wrong.

**Comment by brianlovin** [comment]
Agreed 100% – I just tried to add a new favorite and our search isn't great, and you should be able to add things directly using the `owner/repo` syntax. Going to +1 our tracking issue for this 😊
